### ISSUE TITLE
Email addresses appear to be looked into by name, rather than by memberid

### ISSUE THREAD

**Issue by LordAro** [UniversityRadioYork/MyRadio#588]
Programming team just tried to send an email to memberid 10065, but managed to try to send it to member 7237 instead, who happens to have the same name.

Not yet looked into beyond that, but presumably something's doing a lookup by name somewhere.

Erk.

**Comment by AJDurant** [comment]
Did they look up the members email by searching the list and finding the
wrong person?

**Comment by LordAro** [comment]
No, the wrong person was added to the show, so when the "your show has been added" (or whatever) email got sent, it went to the "wrong" person who has since graduated and their account deactivated (which generated an email bounce)
